docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_remove-ignore-kwarg.txt

```diff
@@ -0,0 +1,4 @@
+* Removed deprecated and unused kwarg ```ignore``` from the following functions:
```

Comment:
deprecated _and_ unused huh? makes me feel a little bit sorry for 'ignore'.

Reply:
let's just go with deprecated.

Reply:
I disagree. It really was unused - it even [said so in the docs](https://github.com/SciTools/iris/pull/2690/files#diff-a06be0c6c86eff57f25f1de2a6e34746L467), so "unused" should stay.